R/bayesianStateSpace.R

```diff
@@ -637,26 +638,27 @@ quantInv <- function(distr, value){
 
   p <- ggplot2::ggplot(NULL,ggplot2::aes(x=time,y=mean))+
     ggplot2::geom_ribbon(mapping=ggplot2::aes(ymin=ymin,ymax =ymax ),
-                         alpha=0.5,fill ="blue") + ggplot2::xlab("Time") +
-    ggplot2::ylab("Distribution") +
+                         alpha=0.5,fill ="blue") + ggplot2::xlab(gettext("Time")) +
+    ggplot2::ylab(gettext("Distribution")) +
     ggplot2::geom_line(size=0.7) +
     ggplot2::theme_classic() +
     ggplot2::geom_hline(yintercept=ul_state, linetype="dashed", color = "red") +
     ggplot2::geom_hline(yintercept=ll_state, linetype="dashed", color = "red")
 
-  if(options$time0 !="")
-    time_date <- "date"
-  else
-    time_date <- "time point"
-
   first_threshold <- time[which(ymin > ul_state)[1]]
 
     p <- jaspGraphs::themeJasp(p)
 
-  if(!is.na(first_threshold))
-    p <- p + ggplot2::labs(caption=paste0("First ",time_date," where ",L,"*sigma is exeeded by ",CI, " % state credible interval: ",first_threshold))
-  else
-    p <- p + ggplot2::labs(caption= paste0("Threshold never reached"))
+  if (!is.na(first_threshold)) {
+    captionText <- if (options$time0 != "")
+      gettextf("First date where %1$.3f*sigma is exceeded by %2$.3f%% state credible interval: %3$s", L, CI, first_threshold)
+    else
+      gettextf("First time point where %1$.3f*sigma is exceeded by %2$.3f%% state credible interval: %3$s", L, CI, first_threshold)
```

Comment:
```suggestion
      gettextf("First date where %1$.3f*sigma is exceeded by %2$.3f%% state credible interval: %3$s", L, CI * 100, first_threshold)
    else
      gettextf("First time point where %1$.3f*sigma is exceeded by %2$.3f%% state credible interval: %3$s", L, CI * 100, first_threshold)
```

Maybe we should multiplied CI by 100 because CI from above is decimal like `.95`. then `0.950%` ->  `95.000%`.